### PR TITLE
JS build fixups

### PR DESCRIPTION
**Pull request by scjody**
@chriddyp Does this look OK?  Can you please test it on Heroku?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "dashboardsly",
+  "version": "0.0.2",
+  "description": "simple websocket communication module",
```
Comment (kiasaki) [inline package.json]: We could maybe put in a description related to Dashboards.ly

**Comment by chriddyp** [comment]
looks good @scjody ! I'll disable auto-deploy, merge, and then manually deploy.

**Comment by scjody** [comment]
@chriddyp Great, thanks.  If it doesn't work let me know and I can work on it more.

**Comment by chriddyp** [comment]
wow, heroku test-apps on PRs! here's a test deployment: https://dashboardsly-pr-14.herokuapp.com/

**Comment by scjody** [comment]
Nice!

**Pull request merged**